src/main/java/com/spotify/reaper/storage/PostgresStorage.java

```diff
@@ -488,4 +489,43 @@ public boolean updateRepairSchedule(RepairSchedule newRepairSchedule) {
       return getPostgresStorage(h).getClusterScheduleOverview(clusterName);
     }
   }
+
+  @Override
+  public boolean takeLeadOnSegment(long segmentId) {
+    return true;
+  }
+
+  @Override
+  public boolean renewLeadOnSegment(long segmentId) {
+    return true;
+  }
+
+  @Override
+  public void releaseLeadOnSegment(long segmentId) {
+    // Not supported with Postgres/H2
+  }
+
+  @Override
+  public void storeHostMetrics(HostMetrics hostMetrics) {
+    // Not supported with Postgres/H2    
+  }
+  
+  @Override
+  public Optional<HostMetrics> getHostMetrics(String hostName) {
+    return Optional.absent();
+  }
+  
+  @Override
+  public StorageType getStorageType() {
+    return StorageType.DATABASE;
+  }
+
+  @Override
+  public int countRunningReapers() {
+    return 1;
+  }
+
+  @Override
+  public void saveHeartbeat() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)